scripts/flexmalloc.sh

```diff
@@ -37,10 +40,27 @@ if [[ "${mpi_rank}" == "0" || "${mpi_rank}" == "" ]] ; then
 	if [[ "${FLEXMALLOC_VERBOSE}" == "" ]] ; then
 		export FLEXMALLOC_VERBOSE=1
 	fi
-	LD_PRELOAD=${FLEXMALLOC_HOME}/lib/${library}.so ${@:3}
-else 
+	# Ease the starting of a program using flexmalloc through GDB debugger
+	if test -n "${FLEXMALLOC_GDB}"
+	then
+		tmp_gdb=`mktemp /tmp/flexmalloc_gdb_cmd_file.XXXXXX`
```

Comment:
May I suggest to use $TMPDIR, if the env variable is set -- otherwise use /tmp.

Reply:
Sure! Good point!